src/ansible_navigator/actions/settings.py

```diff
@@ -0,0 +1,201 @@
+"""The ``settings`` subcommand action."""
+
+import curses
+
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Tuple
+
+from .._yaml import human_dump
+from ..action_base import ActionBase
+from ..action_defs import RunStdoutReturn
+from ..app_public import AppPublic
+from ..configuration_subsystem import transform_settings
+from ..steps import Step
+from ..ui_framework import CursesLinePart
+from ..ui_framework import CursesLines
+from ..ui_framework import Interaction
+from . import _actions as actions
+from . import run_action
+
+
+def filter_content_keys(obj: Dict[Any, Any]) -> Dict[Any, Any]:
+    """Filter out some keys when showing content.
```

Comment:
Reading the function body, these "some keys" don't seem random. So why not use a more accurate description? Maybe
```suggestion
    """Strip private keys off of a dictionary.
```
?

Also, there's nothing in the function body that checks for showing the content so it doesn't need to be mentioned. Maybe it's true on the calling side but definitely not inside the function.

Reply:
This description was approved in https://github.com/ansible/ansible-navigator/pull/880 and is consistent across all actions.

If a change is needed it should happen in a seperate Pr acorss all acitons.

Reply:
I definitely see the value in this. We can make a change for all relevant actions separately.